### ISSUE TITLE
Add room in backpacks to carry ACRE manpack radios

### ISSUE THREAD

**Issue by Drofseh** [Drofseh/IFA3_ACRE2_COMPAT#3]
See title, putting it here so I don't forget.

**Comment by ElTyranos** [comment]
\ww2\Assets_c\Weapons\Misc_c\AdvancedCombatRadioEnvironment2RadioItems_Compatibility\config.cpp

```
B_LIB_GER_Radio_ACRE2
B_LIB_SOV_RA_Radio_ACRE2
B_LIB_US_Radio_ACRE2

maximumLoad = 125;
```

**Comment by Drofseh** [comment]
closed in v0.1.0
